build/scripts/windows-sdk.ps1

```diff
@@ -0,0 +1,109 @@
+#
+# This script launches sdksetup.exe from a folder under TEMP to mitigate potential conflicts with Visual Studio 2015.
+#
+
+param(
+    # TODO: Use version
+    [string]$SdkVersion = $null
+)
+
+$ErrorActionPreference = "Stop"
+
+
+
+function Install-EXE
+{
+    Param
+    (
+        [String]$Url,
+        [String]$Name,
+        [String[]]$ArgumentList
+    )
+
+    $exitCode = -1
+
+    try
+    {
+        Write-Host "Downloading $Name..."
+        $FilePath = "${env:Temp}\$Name"
+
+        Invoke-WebRequest -Uri $Url -OutFile $FilePath
+
+        Write-Host "Starting Install $Name..."
+        $process = Start-Process -FilePath $FilePath -ArgumentList $ArgumentList -Wait -PassThru
+        $exitCode = $process.ExitCode
+
+        if ($exitCode -eq 0 -or $exitCode -eq 3010)
```

Comment:
Nonblocking:
3010 seems to mean ERROR_SUCCESS_REBOOT_REQUIRED. 
1) Would be nice to add a comment about that.
2) Are we sure that without reboot the Win SDK would work fine after install? If not, consider logging a warning in case of 3010, before we proceed?

Reply:
This is copy and pasted from https://microsoft.visualstudio.com/EngSys/_git/pkges.buildvnext.support?path=/Docker/WindowsUndocked/ltsc2019vs2022/windows-sdk.ps1&_a=contents&version=GBmain

I would prefer not to change anything in regards to this file